File: trunk/jtemplate/benchmarks/gcd.js
var gcd=function(x,y){
	return x<y ? gcd(x,y-x) : x>y ? gcd(x-y,x) : x; 
};

var a=0;
for (i=0;i<10000;++i)
	a=gcd(28388383,100101);
print(a);

File: jtemplate/benchmarks/gcd.py
def gcd(a,b):
	if a > b:
		return gcd(a-b,b)
	else:
		if a<b :
			return gcd(a,b-a)
		else :
			return a

a=0

for i in range(1,10000):
	a=gcd(28388383,100101)
	
print(a)	